pkg/operator/operator.go

```diff
@@ -85,15 +80,6 @@ func (o *Operator) Run() error {
 		return fmt.Errorf("Rook operator namespace is not provided. Expose it via downward API in the rook operator manifest file using environment variable %s", k8sutil.PodNamespaceEnvVar)
 	}
 
-	for {
```

Comment:
@ichekrygin what error do we get if the operator is started with an old yaml that doesn't register the CRDs? Does the operator show a decent error in the logs at a later point that makes it obvious the CRDs aren't registered?

Reply:
we could add a better check here -- i.e. check if the CRDs are created and show a better error if they are not.